proposals/simd/SIMD.md

```diff
@@ -104,6 +104,26 @@ operations. This means that any subnormal operand is treated as 0, and any
 subnormal result is rounded to 0. Note that this differs from WebAssembly
 scalar floating-point semantics which require correct subnormal handling.
 
+# JavaScript API and SIMD Values
+
+Accessing WebAssembly module imports or exports containing SIMD Type from JavaScript will throw.
+
+### Module Function Imports
+
+Calling an imported function from JavaScript when the function arguments or result is of type v128 will cause the host function to immidiately throw a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
+
+### Module Global Imports
```

Comment:
Global imports and exports should be legal in the same way as they are for i64 types (after we merged the mutable global proposal). It's only an error when instantiating if it the import object's value is not a `WebAssembly.Global`.

Reply:
Fixed.